app.js

```diff
@@ -86,3 +88,19 @@ function getURLParameter( name ) {
         ( new RegExp( name + '=' + '(.+?)(&|$)' ).exec( location.search ) || [ null, null ] )[ 1 ]
     );
 }
+
+// to facilitate developing print-specific issues
```

Comment:
app.js is not included in the Enketo Core module (when installing from npm), so I thought it was okay and helpful to add this function for us developer. We can do `printView(false)` and `printView(true)` in the console to switch between screen and print views.